lib/upgrade/etcd/conversionv1v3/workloadendpoint_test.go

```diff
@@ -343,6 +345,40 @@ func TestCanConvertV1ToV3WorkloadEndpoint(t *testing.T) {
 	}
 }
 
+func TestBadK8sWorkloadID(t *testing.T) {
+	t.Run("Test invalid k8s workloadID (no dot in name) fails to convert", func(t *testing.T) {
+		RegisterTestingT(t)
+
+		w := conversionv1v3.WorkloadEndpoint{}
+		wepBackendV1 := &model.KVPair{
+			Key: model.WorkloadEndpointKey{
+				Hostname:       "TestNode",
+				OrchestratorID: "k8s",
+				WorkloadID:     "default/frontend-5gs43",
```

Comment:
this example has no `.`s, but I'm guessing one with 3 or more `.`s will also fail

Reply:
no, >1 dot is fine.   Format is <namespace>.<pod>

Only the namespace cannot have a dot in it, but I believe the pod name can.

Reply:
ah ok I missed the part where `SplitN` is set to only 2 matches